test_stopwatch_global.py

```diff
@@ -4,20 +4,38 @@
 
 import pytest
 
+from mock import Mock
+
 from stopwatch_global import (
     global_sw,
     global_sw_del,
     global_sw_init,
 )
 
+
 @pytest.fixture
 def global_sw_fixture(request):
-    global_sw_init()
     request.addfinalizer(global_sw_del)
 
+
 @pytest.mark.usefixtures('global_sw_fixture')
 class TestStopWatchGlobal(object):
+    @staticmethod
+    def add_spans():
+        with global_sw().timer('parent', start_time=20, end_time=80):
+            global_sw().add_span_annotation('parent_annotation', 1)
+            with global_sw().timer('child', start_time=40, end_time=40):
+                global_sw().add_span_annotation('child_annotation', 1)
+
+    def test_reported_traces(self):
+        tracing_function = Mock()
+        global_sw_init(export_tracing_func=tracing_function)
+        self.add_spans()
+        reported_traces = global_sw().get_last_trace_report()
```

Comment:
I think it's important to verify that reported_traces includes 'parent_annotation' and 'child_annotation' as expected.

This test would pass if reported_traces is empty, so it doesn't get to the crux of what we are trying to defend against with our testing.